README.md

```diff
@@ -1,65 +1,91 @@
 # VS Code probe-rs-debugger
 
-
 ## Documentation
-Full documentation on Installation, Configuration and supported functionality can be found at [the probe-rs webpage](https://probe.rs/docs/tools/vscode/) and under the [visual tour heading](https://probe.rs/docs/tools/vscode/#a-visual-guide-of-implemented-features)
 
-![<img src="images/probe-rs-debugger.gif" style="margin-top: 1em; margin-bottom: 1em; max-width:100%; max-height:100%; width: auto; height: auto;"  />](images/probe-rs-debugger.gif)  
+Full documentation on Installation, Configuration and supported functionality
+can be found at [the probe-rs webpage](https://probe.rs/docs/tools/vscode/) and
+under the [visual tour
+heading](https://probe.rs/docs/tools/vscode/#a-visual-guide-of-implemented-features)
+
+![<img src="images/probe-rs-debugger.gif" style="margin-top: 1em; margin-bottom: 1em; max-width:100%; max-height:100%; width: auto; height: auto;"  />](images/probe-rs-debugger.gif)
 
 ## Development Setup
 
-To work on this extensions, you first need to install VS Code and nodejs. Afterwards, follow the following steps:
-
-- Install yarn:
-  ```bash
-  npm install -g yarn
-  ```
-- Checkout this repository
-- Inside the repository, install the prerequisites:
-  ```bash
-  yarn
-  ```
-- Install the extensions VS Code recommends. If you prefer to do this manually, you can find the list of recommended extensions in the repository's `.vscode/settings.json' file. These can then be installed from the command line, for example: 
-  ```bash
-  code --install-extension amodio.tsl-problem-matcher
-  ```
-- Open VS Code
-- Press F5 to start a new VS Code instance where the extension can be debugged. You can also open the "Run and Debug" panel in the left sidebar, and then start the "Extension" debug configuration.
+To work on this extensions, you first need to install VS Code and nodejs.
+Afterwards, follow the following steps:
+
+* Install yarn:
```

Comment:
By removing the ``` modifiers from lines like this, it removes the affordances in webview that allows users to easily copy these as lines of code. Is there a particular reason why you removed them all?

Reply:
Uhh, that's a mistake on my side, I'll try to fix this. (Seems like just 4 chars of indentation is not sufficient when part of a list.)

Reply:
@dbrgn Can you please add the ``` back, so that users an get the clipboard functions to copy these lines for re-use?

Reply:
Ah, now I get what you mean. I personally don't find that important at all (you can just double-click the text, then copy), and it makes the non-rendered version of the markdown file harder to read, but I'll revert this change if you prefer the triple-backtick version!

Reply:
For a short moment I thought that the "copy" icon only appears with the triple-backtick version, but that doesn't seem to be the case.

Here's the rendered version: https://github.com/probe-rs/vscode/tree/prettier

![image](https://user-images.githubusercontent.com/105168/210169408-2e5b815e-246a-44c6-823e-0c2bc8a0c5e9.png)

But maybe I'm misunderstanding you 🙂

Reply:
> I thought that the "copy" icon only appears with the triple-backtick version, but that doesn't seem to be the case.

@dbrgn, it depends. In web view, it shows up, but in VSCode Preview of MD it doesn't show up.  I'll accept this as is, even though I don't understand why prettier insists on taking away the triple-backtick.